src/App.vue

```diff
@@ -91,8 +91,10 @@ export default {
       displayCloseButton: false,
       initial: "Rat",
       helpMode: false,
+      // flatmapAPI: "https://mapcore-demo.org/current/flatmap/v2/"
+      flatmapAPI: "https://mapcore-demo.org/devel/flatmap/v3/"
```

Comment:
Is the v3 for testing or required? What happen if v1 (current production) is used?

Reply:
Yeah, v3 is needed, the previous maps are not annotated with `soma` and `axons` which I need for the tooltip.

if v2 is used, the tooltips get stuck while loading. I will put an update in to fix this